web/src/left_sidebar_navigation_area.js

```diff
@@ -52,6 +54,31 @@ export function deselect_top_left_corner_items() {
     remove($(".top_left_inbox"));
 }
 
+function expand() {
```

Comment:
A future improvement here would be to just add/remove the `collapsed` class and better handle the icon logic from CSS. But I don't see any benefit to fixing that now. Just something to note.

Reply:
Sure once this PR gets merged, will create a follow-up PR that will handle the logic using CSS for this and also for the icon for direct messages as currently both same share logic.